test/token/ERC20/ERC20.test.js

```diff
@@ -287,6 +287,46 @@ contract('ERC20', function (accounts) {
     });
   });
 
+  describe('_update', function () {
+    const amount = 1;
+    
+    it('from is the zero address', async function () {
+      const totalSupply = await this.token.totalSupply();
+      const newSupply = totalSupply.add(new BN(amount));
+      await this.token.update(ZERO_ADDRESS, recipient, amount);
+
+      expect(await this.token.totalSupply()).to.be.bignumber.equal(newSupply);
+      expect(await this.token.balanceOf(recipient)).to.be.bignumber.equal(amount);
+    });
+
+    it('to is the zero address', async function () {
+      const totalSupply = await this.token.totalSupply();
+      const burnAmount = await this.token.balanceOf(initialHolder);
+      await this.token.update(initialHolder, ZERO_ADDRESS, burnAmount);
+
+      expect(await this.token.totalSupply()).to.be.bignumber.equal(totalSupply.sub(burnAmount));
+      expect(await this.token.balanceOf(initialHolder)).to.be.bignumber.equal(new BN(0));
+    });
+
+    describe('from and to are the zero address', function () {
```

Comment:
Please convert this `describe` block into an `it` that looks like the other two cases. I can't see a reason why there should be a separate test for the event.

Reply:
Fixed in last commit.